Article/Article.js

```diff
@@ -109,6 +109,92 @@ const data = [
 
   Step 4: Map over the data, creating a component for each oject and add each component to the DOM as children of the 'articles' div.
 
-  Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new artible
+  Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new article.
 
 */
+
+
+
+
+function thirdParagraphCreator(thirdParagraph){
+  const thirdP = document.createElement('p')
+  thirdP.classList.add('paragraph')
+  thirdP.textContent = thirdParagraph
+  return thirdP
+}
+function secondParagraphCreator(secondParagraph){
+  const secondP = document.createElement('p')
+  secondP.classList.add('paragraph')
+  secondP.textContent = secondParagraph
+  return secondP
+}
+function firstParagraphCreator(firstParagraph){
+  const firstP = document.createElement('p')
+  firstP.classList.add('paragraph')
+  firstP.textContent = firstParagraph
+  return firstP
+}
+//
+function dateParagraphCreator(date){
+  const dateParagraph = document.createElement('p')
+  dateParagraph.classList.add('date')
+  dateParagraph.textContent = date
+  return dateParagraph
+}
+function h1Creator(title){
+  const h1 = document.createElement('h1')
+  h1.textContent = title
+  return h1
+}
+
+const articleContainer = document.querySelector('.articles')
+
+
+function profileCreator (title, date, firstParagraph, secondParagraph, thirdParagraph){
+const profiles = document.createElement('div')
+profiles.classList.add("article")
+profiles.classList.add("close")
+profiles.classList.add("article-open")
+profiles.appendChild(h1Creator(title))
+profiles.appendChild(dateParagraphCreator(date))
+profiles.appendChild(firstParagraphCreator(firstParagraph))
+profiles.appendChild(secondParagraphCreator(secondParagraph))
+profiles.appendChild(thirdParagraphCreator(thirdParagraph))
+
+return profiles
+}
+
+function expandButtonCreator(){
+  const span = document.createElement('span')
+  span.classList.add('expandButton')
+  span.textContent = "open"
+  // const articleOpen = document.querySelectorAll(".article-open")
+  span.addEventListener('click', (event) => {
```

Comment:
Very good implementation of JS logic here!  It looks fantastic!  I think there is a classlist toggle function you can use though.  Basically just classList.toggle('someClass')